test/Repository.js

```diff
@@ -19,9 +19,11 @@ log.level = "silent";
 
 describe("Repository", () => {
   let testDir;
+  let testDirWithWorkspaces;
 
   beforeAll(async () => {
     testDir = await initFixture("Repository/basic");
+    testDirWithWorkspaces = await initFixture("Repository/yarn-workspaces");
```

Comment:
I think we can move this into the one test case that actually needs it, instead of copying over the directory fixture every single test.

```patch
-    it("returns workspace packageConfigs", () => {
+    it("returns workspace packageConfigs", async () => {
+      const testDirWithWorkspaces = await initFixture("Repository/yarn-workspaces");
       const repo = new Repository(testDirWithWorkspaces);
```